src/pvrclient-mythtv.cpp

```diff
@@ -2581,22 +2581,27 @@ void PVRClientMythTV::FillRecordingAVInfo(MythProgramInfo& programInfo, Myth::St
 
 time_t PVRClientMythTV::GetRecordingTime(time_t airtt, time_t recordingtt)
 {
-  if (!g_bUseAirdate || airtt == 0)
-    return recordingtt;
```

Comment:
You shouldn't remove these lines. They are perfect at this place: the recording start time have to be filled as is (time_t).